File: Crassus.Native.iOS/TableViewSourceDelegate.m
//
//  TableViewSourceDelegate.m
//  TheObiG LLC
//
//  Created by SolomonRain on 2/15/16.
//  Copyright © 2016 Crassus. All rights reserved.
//

#import "Crassus_Native_iOS-Swift.h"

void * CrassusCreateTableViewSource(TableViewSource source)
{
    return (void *)CFBridgingRetain([CrassusTableViewSource create: source]);
}

File: Crassus.Native.iOS/TableViewSourceDelegate.c
//
//  TableViewSourceDelegate.c
//  TheObiG LLC
//
//  Created by SolomonRain on 2/15/16.
//  Copyright © 2016 Crassus. All rights reserved.
//

#include "TableViewSourceDelegate.h"

void * CreateTableViewSource(TableViewSource source, TableViewSourceCreator function)
{
    return (*function)(source);
}
